modules/delegates.js

```diff
@@ -52,6 +52,8 @@ function Delegates (cb, scope) {
 		config: {
 			forging: {
 				secret: scope.config.forging.secret,
+				force: scope.config.forging.force,
+				defaultKey: scope.config.forging.defaultKey,
```

Comment:
Ok got it, so for `testnet`, `devnet` purposes. Therefore assuming users use this on a `mainnet` node, it defeats whole purpose of encrypting passphrases in the first place.